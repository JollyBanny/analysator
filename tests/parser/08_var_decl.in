var a, b: integer;
    str: string;
begin
end.