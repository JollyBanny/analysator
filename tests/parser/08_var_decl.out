program
└──── program block
      ├──── VAR
      │     ├──── var declaration
      │     │     ├──── a, b
      │     │     └──── integer
      │     └──── var declaration
      │           ├──── str
      │           └──── STRING
      └──── Compound statement
            └──── Empty statement
